tutorials/v7/draw_text.cxx

```diff
@@ -33,10 +33,14 @@ void draw_text()
    auto canvas = RCanvas::Create("Canvas Title");
 
    for (int i=0; i<=360; i+=10) {
-      auto text = canvas->Draw<RText>(RPadPos(0.5_normal, 0.6_normal), "____  Hello World");
+      auto draw = canvas->Draw<RText>(RPadPos(0.5_normal, 0.6_normal), "____  Hello World");
 
       RColor col((int) (0.38*i), (int) (0.64*i), (int) (0.76*i));
-      text->AttrText().SetColor(col).SetSize(10+i/10).SetAngle(i).SetAlign(13).SetFont(42);
+      draw->text.color = col;
+      draw->text.size = 10+i/10;
+      draw->text.angle = i;
+      draw->text.align = 13;
+      draw->text.font = 4;
```

Comment:
```suggestion
      draw->text = {
         .color = col,
         .size = 10+i/10,
         .angle = i,
         .align = 13,
         .font = 4
      };
```

Reply:
Or maybe even better:
```
for (int i=0; i<=360; i+=10) {
   RAttrText attr{
      .color = RColor((int) (0.38*i), (int) (0.64*i), (int) (0.76*i)),
      .size = 10+i/10,
      .angle = i,
      .align = 13,
      .font = 4
   };
   canvas->Draw<RText>(RPadPos(0.5_normal, 0.6_normal), "____  Hello World", attr);
}

Reply:
Unfortunately, does not work with `RAttr` classes. And even then - such initializer lists not works on Windows with simplest class. See https://godbolt.org/z/TGMoqv8cP

Reply:
As discussed on mattermost, MSVC needs `/std:c++20` or `/std:c++latest` and the `RAttr` classes must not have a user provided constructor. For gcc/clang you can rely on designated initializers as a vendor extension.

I still want to emphasis that even though the currently designed API might not be usable now with designated initializers in C++14 cross platform, it will in the future. And I am pretty sure that this API will live long enough to see such a use.